modules/oe_webtools_globan/oe_webtools_globan.module

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @file
+ * Contains Drupal hooks.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Asset\AttachedAssetsInterface;
+use Drupal\Core\Cache\CacheableMetadata;
+
+/**
+ * Implements hook_page_attachments().
+ *
+ * Attach smart loader.
+ */
+function oe_webtools_globan_page_attachments(array &$attachments): void {
```

Comment:
Please check that we do actually need this cache meta

Reply:
Confirmed. Without this cache meta, cache invalidation doesn't work correctly.